### ISSUE TITLE
Model does not match

### ISSUE THREAD

**Issue by edwardtfn** [Blackymas/NSPanel_HA_Blueprint#1310]
I think I'm seing the same thing as @hansiputz.

`Nextion is not connected` is everywhere in the logs.
Also the diagnostic in HA says: `Nextion display: Disconnected`

The log capture during my upload is:
```
[17:41:38][D][nextion_upload:145]: Nextion TFT upload requested
[17:41:38][D][nextion_upload:146]: url: http://192.168.178.63:8123/local/nspanel_eu.tft
[17:41:38][D][nextion_upload:185]: Connected
[17:41:38][D][nextion_upload:191]: Requesting URL: http://192.168.178.63:8123/local/nspanel_eu.tft
[17:41:38][D][nextion_upload:225]: Updating Nextion NX4832F035_011C...
[17:41:38][D][nextion_upload:251]: Waiting for upgrade response
[17:41:39][D][nextion_upload:257]: Upgrade response is [05] - 1 bytes
[17:41:39][D][nextion_upload:260]: Available 0 : 0x05
[17:41:39][D][nextion_upload:264]: preparation for tft update done
[17:41:39][D][nextion_upload:292]: Allocating buffer size 65536, Heap size is 175452
[17:41:39][D][nextion_upload:309]: Updating tft from "http://192.168.178.63:8123/local/nspanel_eu.tft" with a file size of 137052 using 65536 chunksize, Heap Size 109868
[17:41:39][D][nextion_upload:049]: Requesting range: bytes=0-16383
[17:41:39][D][nextion_upload:113]: Uploaded 2.99 %, remaining 132956 bytes
[17:41:44][D][nextion_upload:123]: recv_string []
[17:41:44][D][nextion_upload:113]: Uploaded 5.98 %, remaining 128860 bytes
[17:41:48][D][nextion_upload:123]: recv_string []
[17:41:48][D][nextion_upload:113]: Uploaded 8.97 %, remaining 124764 bytes
[17:41:53][D][nextion_upload:123]: recv_string []
[17:41:53][D][nextion_upload:113]: Uploaded 11.95 %, remaining 120668 bytes
[17:41:57][D][nextion_upload:123]: recv_string []
[17:41:57][D][nextion_upload:320]: Heap Size 111592, Bytes left 120668
[17:41:57][D][nextion_upload:049]: Requesting range: bytes=16384-81919
[17:41:58][D][nextion_upload:113]: Uploaded 14.94 %, remaining 116572 bytes
[17:42:02][D][nextion_upload:123]: recv_string []
[17:42:02][D][nextion_upload:113]: Uploaded 17.93 %, remaining 112476 bytes
[17:42:06][D][nextion_upload:123]: recv_string []
[17:42:07][D][nextion_upload:113]: Uploaded 20.92 %, remaining 108380 bytes
[17:42:11][D][nextion_upload:123]: recv_string []
[17:42:11][D][nextion_upload:113]: Uploaded 23.91 %, remaining 104284 bytes
[17:42:15][D][nextion_upload:123]: recv_string []
[17:42:15][D][nextion_upload:113]: Uploaded 26.90 %, remaining 100188 bytes
[17:42:20][D][nextion_upload:123]: recv_string []
[17:42:20][D][nextion_upload:113]: Uploaded 29.89 %, remaining 96092 bytes
[17:42:24][D][nextion_upload:123]: recv_string []
[17:42:24][D][nextion_upload:113]: Uploaded 32.88 %, remaining 91996 bytes
[17:42:28][D][nextion_upload:123]: recv_string []
[17:42:29][D][nextion_upload:113]: Uploaded 35.86 %, remaining 87900 bytes
[17:42:33][D][nextion_upload:123]: recv_string []
[17:42:33][D][nextion_upload:113]: Uploaded 38.85 %, remaining 83804 bytes
[17:42:37][D][nextion_upload:123]: recv_string []
[17:42:38][D][nextion_upload:113]: Uploaded 41.84 %, remaining 79708 bytes
[17:42:42][D][nextion_upload:123]: recv_string []
[17:42:42][D][nextion_upload:113]: Uploaded 44.83 %, remaining 75612 bytes
[17:42:46][D][nextion_upload:123]: recv_string []
[17:42:47][D][nextion_upload:113]: Uploaded 47.82 %, remaining 71516 bytes
[17:42:51][D][nextion_upload:123]: recv_string []
[17:42:51][D][nextion_upload:113]: Uploaded 50.81 %, remaining 67420 bytes
[17:42:55][D][nextion_upload:123]: recv_string []
[17:42:56][D][nextion_upload:113]: Uploaded 53.80 %, remaining 63324 bytes
[17:43:00][D][nextion_upload:123]: recv_string []
[17:43:00][D][nextion_upload:113]: Uploaded 56.78 %, remaining 59228 bytes
[17:43:04][D][nextion_upload:123]: recv_string []
[17:43:04][D][nextion_upload:113]: Uploaded 59.77 %, remaining 55132 bytes
[17:43:09][D][nextion_upload:123]: recv_string []
[17:43:09][D][nextion_upload:320]: Heap Size 104316, Bytes left 55132
[17:43:09][D][nextion_upload:049]: Requesting range: bytes=81920-137052
[17:43:09][D][nextion_upload:113]: Uploaded 62.76 %, remaining 51036 bytes
[17:43:13][D][nextion_upload:123]: recv_string []
[17:43:14][D][nextion_upload:113]: Uploaded 65.75 %, remaining 46940 bytes
[17:43:18][D][nextion_upload:123]: recv_string []
[17:43:18][D][nextion_upload:113]: Uploaded 68.74 %, remaining 42844 bytes
[17:43:22][D][nextion_upload:123]: recv_string []
[17:43:22][D][nextion_upload:113]: Uploaded 71.73 %, remaining 38748 bytes
[17:43:27][D][nextion_upload:123]: recv_string []
[17:43:27][D][nextion_upload:113]: Uploaded 74.72 %, remaining 34652 bytes
[17:43:31][D][nextion_upload:123]: recv_string []
[17:43:31][D][nextion_upload:113]: Uploaded 77.70 %, remaining 30556 bytes
[17:43:35][D][nextion_upload:123]: recv_string []
[17:43:36][D][nextion_upload:113]: Uploaded 80.69 %, remaining 26460 bytes
[17:43:40][D][nextion_upload:123]: recv_string []
[17:43:40][D][nextion_upload:113]: Uploaded 83.68 %, remaining 22364 bytes
[17:43:44][D][nextion_upload:123]: recv_string []
[17:43:45][D][nextion_upload:113]: Uploaded 86.67 %, remaining 18268 bytes
[17:43:49][D][nextion_upload:123]: recv_string []
[17:43:49][D][nextion_upload:113]: Uploaded 89.66 %, remaining 14172 bytes
[17:43:53][D][nextion_upload:123]: recv_string []
[17:43:54][D][nextion_upload:113]: Uploaded 92.65 %, remaining 10076 bytes
[17:43:58][D][nextion_upload:123]: recv_string []
[17:43:58][D][nextion_upload:113]: Uploaded 95.64 %, remaining 5980 bytes
[17:44:02][D][nextion_upload:123]: recv_string []
[17:44:03][D][nextion_upload:113]: Uploaded 98.63 %, remaining 1884 bytes
[17:44:07][D][nextion_upload:123]: recv_string []
[17:44:07][D][nextion_upload:113]: Uploaded 100.00 %, remaining 0 bytes
[17:44:11][D][nextion_upload:123]: recv_string []
[17:44:11][D][nextion_upload:320]: Heap Size 88532, Bytes left 0
[17:44:11][D][nextion_upload:322]: Successfully updated Nextion!
[17:44:11][D][nextion_upload:329]: Restarting Nextion
WARNING nspanel-livingroom: Connection error occurred: [Errno 104] Connection reset by peer
INFO Processing unexpected disconnect from ESPHome API for nspanel-livingroom
WARNING Disconnected from API
INFO Successfully connected to nspanel-livingroom in 0.006s
INFO Successful handshake with nspanel-livingroom in 0.039s
[17:44:32][W][nextion:077]: Nextion is not connected! 
[17:44:33][W][nextion:077]: Nextion is not connected! 
[17:44:33][W][nextion:077]: Nextion is not connected! 
[17:44:34][W][nextion:077]: Nextion is not connected! 
[17:44:34][W][nextion:077]: Nextion is not connected! 
[17:44:35][W][nextion:077]: Nextion is not connected! 
[17:44:36][W][nextion:077]: Nextion is not connected! 
[17:44:36][W][nextion:077]: Nextion is not connected! 
[17:44:37][W][nextion:077]: Nextion is not connected! 
[17:44:37][W][nextion:077]: Nextion is not connected! 
[17:44:38][W][nextion:077]: Nextion is not connected! 
[17:44:38][W][nextion:077]: Nextion is not connected! 
[17:44:39][W][nextion:077]: Nextion is not connected! 
[17:44:39][W][nextion:077]: Nextion is not connected! 
[17:44:40][W][nextion:077]: Nextion is not connected! 
[17:44:40][D][main:2485]: Exit reparse
[17:44:40][W][nextion:077]: Nextion is not connected! 
[17:44:41][W][nextion:077]: Nextion is not connected! 
[17:44:41][W][nextion:077]: Nextion is not connected! 
[17:44:42][W][nextion:077]: Nextion is not connected! 
[17:44:42][W][nextion:077]: Nextion is not connected! 
[17:44:43][W][nextion:077]: Nextion is not connected! 
[17:44:43][W][nextion:077]: Nextion is not connected! 
[17:44:44][W][nextion:077]: Nextion is not connected! 
[17:44:45][W][nextion:077]: Nextion is not connected! 
[17:44:45][W][nextion:077]: Nextion is not connected! 
[17:44:46][W][nextion:077]: Nextion is not connected! 
[17:44:46][W][nextion:077]: Nextion is not connected! 
[17:44:47][W][nextion:077]: Nextion is not connected! 
[17:44:47][W][nextion:077]: Nextion is not connected! 
[17:44:48][W][nextion:077]: Nextion is not connected! 
[17:44:48][W][nextion:077]: Nextion is not connected! 
[17:44:49][W][nextion:077]: Nextion is not connected! 
[17:44:49][W][nextion:077]: Nextion is not connected! 
[17:44:50][W][nextion:077]: Nextion is not connected! 
[17:44:50][W][nextion:077]: Nextion is not connected! 
[17:44:51][W][nextion:077]: Nextion is not connected! 
[17:44:51][W][nextion:077]: Nextion is not connected! 
[17:44:52][W][nextion:077]: Nextion is not connected! 
[17:44:52][W][nextion:077]: Nextion is not connected! 
[17:44:53][W][nextion:077]: Nextion is not connected! 
[17:44:54][W][nextion:077]: Nextion is not connected! 
[17:44:54][W][nextion:077]: Nextion is not connected! 
[17:44:55][W][nextion:077]: Nextion is not connected! 
[17:44:55][W][nextion:077]: Nextion is not connected! 
[17:44:56][W][nextion:077]: Nextion is not connected! 
[17:44:56][W][nextion:077]: Nextion is not connected! 
[17:44:57][W][nextion:077]: Nextion is not connected! 
[17:44:57][W][nextion:077]: Nextion is not connected! 
[17:44:58][W][nextion:077]: Nextion is not connected! 
[17:44:58][W][nextion:077]: Nextion is not connected! 
[17:44:59][W][nextion:077]: Nextion is not connected! 
[17:44:59][W][nextion:077]: Nextion is not connected! 
[17:45:00][W][nextion:077]: Nextion is not connected! 
[17:45:00][E][on_boot:040]: No response from Nextion display
[17:45:00][D][on_boot:041]: Turn off Nextion
[17:45:00][D][switch:016]: 'nspanel-livingroom Screen Power' Turning OFF.
[17:45:00][D][switch:055]: 'nspanel-livingroom Screen Power': Sending state OFF
[17:45:02][D][on_boot:044]: Turn on Nextion
[17:45:02][D][switch:012]: 'nspanel-livingroom Screen Power' Turning ON.
[17:45:02][D][switch:055]: 'nspanel-livingroom Screen Power': Sending state ON
[17:45:02][W][component:214]: Component esphome.coroutine took a long time for an operation (1.53 s).
[17:45:02][W][component:215]: Components should block for at most 20-30ms.
[17:45:02][W][nextion:077]: Nextion is not connected! 
[17:45:02][I][nextion:083]: Nextion is connected
[17:45:02][C][nextion:123]: Nextion:
[17:45:02][C][nextion:124]:   Device Model:     NX4832F035_011C
[17:45:02][C][nextion:125]:   Firmware Version: 51
[17:45:02][C][nextion:126]:   Serial Number:    B9B03401BE69B630
[17:45:02][C][nextion:127]:   Flash Size:       16777216
[17:45:02][C][nextion:128]:   Wake On Touch:    True
[17:45:02][C][nextion:139]:   Start Up Page :      8
[17:45:02][W][nextion:331]: Nextion reported invalid instruction!
[17:45:02][E][nextion:274]: Nextion queue is empty!
[17:45:02][D][nextion:563]: system successful start up 0
[17:45:02][W][nextion:331]: Nextion reported invalid instruction!
[17:45:02][E][nextion:274]: Nextion queue is empty!
[17:45:02][D][nextion:563]: system successful start up 0
[17:45:02][W][component:214]: Component nextion.display took a long time for an operation (0.19 s).
[17:45:02][W][component:215]: Components should block for at most 20-30ms.
[17:45:10][D][nextion:745]: Removing old queue type "NO_RESULT" name "backlight_brightness"
[17:45:10][D][nextion:745]: Removing old queue type "NO_RESULT" name "goto_page"
[17:45:10][D][nextion:745]: Removing old queue type "NO_RESULT" name "auto_wake_on_touch"
[17:45:13][D][sensor:094]: 'ntc_source': Sending state 1.60200 V with 2 decimals of accuracy
[17:45:13][D][resistance:039]: 'resistance_sensor' - Resistance 10566.8Ω
[17:45:13][D][sensor:094]: 'resistance_sensor': Sending state 10566.78418 Ω with 1 decimals of accuracy
[17:45:13][D][ntc:026]: 'nspanel-livingroom Temperature' - Temperature: 23.8°C
[17:45:13][D][sensor:094]: 'nspanel-livingroom Temperature': Sending state 23.76444 °C with 1 decimals of accuracy
[17:45:22][E][on_boot:054]: No response from Nextion display
[17:45:22][D][on_boot:056]: Finished
```

During upload the NSPanel shows:
```
Model does not match
Device Model:
NX4832F035_011C
```

_Originally posted by @munich-ml in https://github.com/Blackymas/NSPanel_HA_Blueprint/issues/1308#issuecomment-1817913541_

**Comment by edwardtfn** [comment]
@munich-ml, looks like you have an incomplete `nspanel_eu.tft` file in your local server.
Please try download the file from GitHub and upload to your local server again, then try to upload to your display.

**Comment by munich-ml** [comment]
@edwardtfn, you saved my day!

When I downloaded the nspanel_eu.tft file using wget `https://github.com/Blackymas/NSPanel_HA_Blueprint/nspanel_eu.tft` I get a file that is 133 KB large and apparently not complete. With this incomplete file I experienced the issues as discussed above.

However, when I download the nspanel_eu.tft from my Windows PC, it is 7 MB large. Once I used that one, everything worked fine!

Thanks a lot!
